networkx/algorithms/matching.py

```diff
@@ -93,6 +93,9 @@ def max_weight_matching(G, maxcardinality=False):
     paths and the "primal-dual" method for finding a matching of maximum
     weight, both methods invented by Jack Edmonds [1]_.
 
+    Bipartite graphs can also be matched using the functions present in
+    ``networkx.algorithms.bipartite.matching``
```

Comment:
This should be a module reference: `` :mod:`networkx.algorithms.bipartite.matching` ``, and the sentence should end with a period.